index.html

```diff
@@ -1,10 +1,68 @@
+
 <!DOCTYPE html>
 <html lang="en">
     <head>
         <meta charset="UTF-8" />
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
         <title>Kaizen 2021</title>
+        <link rel="stylesheet" href="master.css">
```

Comment:
The files master.css is missing from the project, please see to this.
Also, files must be placed in CSS folder then inside the href you can refer it with "./css/file_name.css".
Lastly run command "npm run prettier-fix" in the root directory inside the terminal , it formats code with proper indentation and spaces.